internal/util/auth/auth.go

```diff
@@ -26,15 +29,30 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+const (
+	authFlowNamedMutex         = "monoctl-auth-flow"
+	namedMutexDetectionTimeOut = 500 * time.Millisecond
+)
+
+var silent bool
+
 func RetryOnAuthFailSilently(ctx context.Context, configManager *config.ClientConfigManager, f func(ctx context.Context) error) error {
-	return retryOnAuthFail(ctx, configManager, true, f)
+	silent = true
```

Comment:
Can you shortly explain why this global variable thing is necessary instead of the way it was before?

Reply:
because of the print function down there to keep the signature the same as the default one, which makes removing it a little easier 🙃, would you suggest passing it instead?